tests/SprykerEcoTest/Zed/AmazonPay/Business/AmazonpayFacadeAddSelectedShipmentMethodToQuoteTest.php

```diff
@@ -39,17 +39,15 @@ protected function createQuote()
      * @dataProvider addSelectedShipmentMethodToQuoteProvider
      *
      * @param \Generated\Shared\Transfer\QuoteTransfer $quoteTransfer
-     * @param int $shipmentMethodIndex
      * @param string $shipmentMethodName
      * @param int $shipmentPrice
      *
      * @return void
      */
-    public function testAddSelectedShipmentMethodToQuote(QuoteTransfer $quoteTransfer, $shipmentMethodIndex, $shipmentMethodName, $shipmentPrice)
+    public function xtestAddSelectedShipmentMethodToQuote(QuoteTransfer $quoteTransfer, $shipmentMethodName, $shipmentPrice)
```

Comment:
Please check do we really need this logic or we can get rid of this and use Shipping client on Yves/Client level